build/scripts/Build.ps1

```diff
@@ -125,7 +128,20 @@ Try {
     }
     if (-not ([string]::IsNullOrEmpty($newAppDisplayNameResource))) {
       $appxmanifest.Root.Element($xApplications).Element($xApplication).Element($uapVisualElements).Attribute("DisplayName").Value = $newAppDisplayNameResource
-      $appxmanifest.Root.Element($xApplications).Element($xApplication).Element($xExtensions).Element($uapExtension).Element($uapAppExtension).Attribute("DisplayName").Value = $newAppDisplayNameResource

```

Comment:
How was this working before? It was updating the `DisplayName` of everything to "DevHome GihHub Extension"? Trying to understand the change to the loop here.

Reply:
Also, we were lucky it was working before.  It was editting the first uapExtension it found which just so happened to be the one we wanted.  If someone added another uapExtension earlier in the manifest, this would have broke.